sorts/bucket_sort.py

```diff
@@ -30,7 +30,7 @@
 from __future__ import annotations
 
 
-def bucket_sort(my_list: list) -> list:
+def bucket_sort(my_list: list, bucket_count: int = 10) -> list:
```

Comment:
very nit-picky / low-priority - 
Can we change variable name to something more descriptive? Maybe have list_to_sort instead of my_list?